src/shipit_pulse_listener/shipit_pulse_listener/cli.py

```diff
@@ -4,11 +4,11 @@
 
 @click.command()
 @click.argument('branch', type=click.Choice(['staging', 'production']))
-@click.option('--secrets', required=True, help='Taskcluster Secrets path')
-@click.option('--client-id', help='Taskcluster Client ID')
-@click.option('--client-token', help='Taskcluster Client token')
-def main(branch, secrets, client_id, client_token):
-    pl = PulseListener(secrets, client_id, client_token)
+@click.option('--taskcluster-secret', help='Taskcluster Secret path')
```

Comment:
this triplets of options is repreating all the time. should we extract this in ``cli_common/click.py``? also i would add to read from ``envvar``.